server/server.go

```diff
@@ -2098,6 +2098,10 @@ func (s *Server) Start() {
 	// Pprof http endpoint for the profiler.
 	if opts.ProfPort != 0 {
 		s.StartProfiler()
+	} else {
+		// It's still possible to access this profile via a SYS endpoint, so set
+		// this anyway. (Otherwise StartProfiler would have called it.)
+		s.setBlockProfileRate(opts.ProfBlockRate)
```

Comment:
I thought this was bad to set by default no?

Reply:
Is it zero by default? Meaning does not set anything penal?

Reply:
Yep, zero by default, which disables the block profiler.